iidm/iidm-api/src/main/java/com/powsybl/iidm/network/limitmodification/result/UnalteredLimitsContainer.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network.limitmodification.result;
+
+/**
+ * @author Olivier Perrin {@literal <olivier.perrin at rte-france.com>}
+ */
+public class UnalteredLimitsContainer<L> implements LimitsContainer<L> {
```

Comment:
Equality? Identical?

Reply:
I finally used `IdenticalLimitsContainer` (and `AbstractDistinctLimitsContainer`, with a `isDistinct()` method).